tests/functional_tests/blockchain.py

```diff
@@ -80,10 +80,10 @@ def _test_generateblocks(self, blocks):
         assert ok
 
         res = daemon.get_fee_estimate()
-        assert res.fee == 234562
+        assert res.fee == 1200000
```

Comment:
I can't see why yet but this is causing a test to fail. Working through it

Reply:
Probably because it's not on v15 yet. You need to add that fork to the table.

Reply:
Yep, adding to the end of the mainnet table does the trick. Added it in the wrong way and was getting a weird result